pypeit/tests/test_syncspec.py

```diff
@@ -5,6 +5,7 @@
 from pypeit.scripts import coadd_1dspec
 from pypeit.tests.tstutils import dev_suite_required, cooked_required
 
+# TODO: Bring this back?
```

Comment:
We could, but it would blow up the Cooked folder yet further..

Reply:
Okay, removing the TODO comment, but leaving the commented code.